### PR TITLE
Added if statement for asset cloudfront alb

### PR DESCRIPTION
**Pull request by robinkwilson**
Investigating missing assets for enterprise multi-server HC stacks.

### CODE REVIEW

**Review by brianpeiris — APPROVED** [review]
LGTM, though we should think about when exactly is the right time to merge this, as we discussed.

**Comment by robinkwilson** [comment]
## QA Test Plan

### TLDR PR context
🐛 🏙️ Enterprise multi-server deployments use round-robin load balancing though ALB (application load balancer) is selected in the template. This fixes that issue.

**_I will combine all of these template changes into one for a thorough hubs test._**

At a minimum for testing just this PR change:
✅ Verify ALB is deployed in AWS

## 🏁 Plan

I will provide template links via email 📫 .

### 🏛️ Update existing stack with a new template (users with existing stacks will follow this workflow)
#### For ONLY Enterprise multi-server 🏙️ 
1. AWS Cloudformation
2. Click on your stack
3. Click "Update"
4. Select "Replace current template"
5. Paste in new template URL
6. Click "Next"
7. ☑️ Ensure Application Load Balancer is selected (🏙️ Enterprise multi-server default)
8. Deploy then follow "Verify below cases work for each deployment"

### ✨ New stacks, create a new stack with template
#### For Enterprise multi-server 🏙️ 
1. AWS Cloudformation
2. Click "Create stack"
3. Paste in new template URL
4. ☑️ Ensure Application Load Balancer is selected (🏙️ Enterprise multi-server default)
5. Deploy then follow "Verify below cases work for each deployment"

### ✅ Verify ALB is deployed in AWS
1. Ensure ALB is selected in deploy stack or update stack with new template
2. TBD
7. Complete! 😄 🎉
